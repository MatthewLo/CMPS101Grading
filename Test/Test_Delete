https://github.com/ShobhitMaheshwari/CMPS101Grading/blob/master/Test/Test_Delete.c
5
Check list insert, and index
gcc -std=c99 -Wall -c Test_Delete.c > w 2>&1
gcc -std=c99 -Wall -o Test_Delete Test_Delete.o ../List.o &> r
./Test_Delete
valgrind --leak-check=full --log-file="memoryLeak" ./Test_Delete
