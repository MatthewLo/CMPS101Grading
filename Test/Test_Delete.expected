0 1 4 9 16 
16 9 4 1 0
1 4 9 16 
4 9 16 
9 16 
16 

16 9 4 1
16 9 4
16 9
16

0 1 4 9 16 
16 9 4 1 0
-1
0 1 9 16 
9 4 1 0
9 4 1
